clang/bindings/python/clang/cindex.py

```diff
@@ -3585,7 +3600,7 @@ def filename(self):
     def arguments(self):
         """
         Get an iterable object providing each argument in the
-        command line for the compiler invocation as a _CXString.
+        command line for the compiler invocation as a string.
```

Comment:
Same as above: this is not supposed to be exposed to the user, and has also been changed previously to return `str` instead of `_CXString`